io/pci/dlpar.py

```diff
@@ -67,6 +67,12 @@ def setUp(self):
         if not self.server:
             self.cancel("Managed System not got")
         self.lpar_2 = self.params.get("lpar_2", default=None)
+        self.pci_device = self.params.get("pci_device", default=None).split(' ')
```

Comment:
rename to pci_devices  for multiple device support

Reply:
I have modified the pci_device to pci_devices in dlpar.py and yaml file